src/store/tools/id.ts

```diff
@@ -0,0 +1,13 @@
+import { defineStore } from 'pinia';
+
+export const useIDStore = defineStore('IDStore', () => {
+  type _This = ReturnType<typeof useIDStore>;
+
+  function nextID(this: _This) {
+    return this.$id.nextID();
```

Comment:
I was actually suggesting we get rid of the `this.$id` injection and just use an ID store, but I think that's out-of-scope. I can follow up with the changes necessary to accomplish this.